src/assets/scripts/client/airport/AirportWind.js

```diff
@@ -0,0 +1,35 @@
+import AirportModel from './AirportModel';
+import { calculateNextWind } from '../utilities/windUtilities';
+
+/**
+ * Mutable array that holds the information on the airport's current wind.
+ */
+export let airportWind = {
+   speed: 10,
+   angle: 0
+};
+
+export default class AirportWind {
+    /**
+     * Constructor
+     */
+    constructor() {}
```

Comment:
so then this line becomes:
```javascript
constructor() {
  this.angle = -9999;
  this.speed = -1;

  return this._init();
}
```

notice we initialize the properties with invalid values.  within the `_init()` method you would then set the default values for those properties.